src/visualizers/widgets/MonacoEditor/styles/MonacoEditorWidget.css

```diff
@@ -0,0 +1,24 @@
+/**
+ * This file is for any css that you may want for this visualizer.
+ *
+ * Ideally, you would use the scss file also provided in this directory
+ * and then generate this file automatically from that. However, you can
+ * simply write css if you prefer
+ */
+
+.monaco-editor {
+    outline: none;
+    padding: 0px;
+    text-align: left !important;
```

Comment:
I would recommend writing these in scss and then generating this file. If you prefer writing the css here, then delete the scss file (though I honestly don't see any good reasons for writing css over scss).